odk/src/main/resources/META-INF/spring/blueprint.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:osgi="http://www.eclipse.org/gemini/blueprint/schema/blueprint"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd http://www.eclipse.org/gemini/blueprint/schema/blueprint http://www.eclipse.org/gemini/blueprint/schema/blueprint/gemini-blueprint.xsd">
+
+    <import resource="classpath*:META-INF/motech/*.xml"/>
+
+    <bean id="moduleRegistrationData" class="org.motechproject.osgi.web.ModuleRegistrationData">
+        <constructor-arg name="url" value="../odk/resources/index.html"/>
+        <constructor-arg name="moduleName" value="odk"/>
+        <constructor-arg name="angularModules">
+            <list>
+                <value>odk</value>
+            </list>
+        </constructor-arg>
+        <constructor-arg name="i18n">
+            <map>
+                <entry key="messages" value="../odk/resources/messages/"/>
+            </map>
+        </constructor-arg>
+        <property name="settingsURL" value="/odk/settings"/>
+        <property name="defaultURL" value="/odk/settings"/>
+    </bean>
+
+
+    <osgi:reference id="httpClientBuilderFactory" interface="org.apache.http.osgi.services.HttpClientBuilderFactory"/>
+    <osgi:reference id="channelService" interface="org.motechproject.tasks.service.ChannelService"/>
+    <osgi:reference id="formDefinitionDataService" interface="org.motechproject.odk.repository.FormDefinitionDataService"/>
+    <osgi:reference id="formFailureDataService" interface="org.motechproject.odk.repository.FormFailureDataService"/>
+    <osgi:reference id="formInstanceDataService" interface="org.motechproject.odk.repository.FormInstanceDataService"/>
+    <osgi:reference id="eventRelay" interface="org.motechproject.event.listener.EventRelay"/>
+    <osgi:reference id="eventListenerRegistryService" interface="org.motechproject.event.listener.EventListenerRegistryService"/>
+
+    <osgi:service ref="eventHandler" interface="org.motechproject.odk.event.EventHandler"  />
+    <osgi:service ref="odkConfigurationService" interface="org.motechproject.odk.service.ConfigurationService"/>
```

Comment:
Why not expose more services? I think implementations might find them usefull down the line